ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/parser/SerialTreeParser.java

```diff
@@ -78,8 +79,14 @@ public Node parse(String source) throws TreeParserException {
                 errorMessage = "Code contains syntax error(s).";
             }
         }
-        addErrorDiagnostic(errorMessage);
-        addErrorDiagnostic("Parsing aborted due to errors.");
+        if (source.startsWith(COMMAND_PREFIX)) {
+            errorMessage = "Can not find the command : " + source.substring(0, source.length() - 1).trim();
```

Comment:
Typically the we use the colon in messages as `<message>: <additional-info>`

Reply:
Updated in 13a87c7ddf4772d1855f9f491949e8e02878ef59